_data/events_gdcr2018/Hyderabad-India-CDKGlobal.json

```diff
@@ -2,9 +2,7 @@
   "title": "Coderetreat @ CDK Global (India) Pvt. Ltd.",
   "url": "https://sites.google.com/view/cdkhyderabadevents",
   "moderators": [
-    "Vibhor Mamgain",
-    "Deepesh Patel",
-    "@patel_deepesh"
+    "Vibhor Mamgain"
```

Comment:
But you mentioned you will be the one running the event in Hyberabad, so, in `moderators` key we should only have your name : `"@patel_deepesh"`